modules/4337/docs/v0.2.0/audit.md

```diff
@@ -16,6 +18,22 @@ There is one acknowledged finding from the audit report:
   > - The most critical functionality, such as signature checks and replay protection, is handled by the Safe and Entrypoint contracts.
   > - The entrypoint contract uses the optimizer.
 
-## Files
+### Files
 
 - [Final audit report](audit-report-v2.0.pdf)
+
+## Audit 2
+
+### Auditor
+
+OpenZeppelin (<https://openzeppelin.com/>).
+
+### Notes
+
+The final audit was performed on commit [3853f34f31837e0a0aee47a4452564278f8c62ba](https://github.com/safe-global/safe-modules/tree/3853f34f31837e0a0aee47a4452564278f8c62ba).
+
+No critical or high severity issues were found. For the detailed findings, please refer to the audit report.
```

Comment:
We should mention that we did not deploy new contracts as a result of the audit review - as there were _no security-related smart contract code changes_.